Zend/tests/in-de-crement/incdec_undefined_vars_exception.phpt

```diff
@@ -0,0 +1,48 @@
+--TEST--
+Inc/dec on undefined variable: warning converted to exception
+--FILE--
+<?php
+
+set_error_handler(function($severity, $m) {
+    throw new Exception($m, $severity);
+});
+
+unset($x);
+try {
+    $x++;
+} catch (\Exception $e) {
+    echo $e->getMessage(), PHP_EOL;
+    if (!isset($x)) { echo("UNDEF\n"); } else { var_dump($x); }
```

Comment:
I just realized that ``isset()`` doesn't make distinct between ``UNDEF`` and ``NULL``, so the test prints UNDEF insted of NULL. It's possible to fix this using the following terrible code.

```php
if (!isset($x) && !@is_null($x)) { echo("UNDEF\n"); } else { var_dump($x); }
```

Reply:
@dstogov I don't think that works, as `$x` is coerced to `NULL`, no? It could be possible to just access `$x` and catch the warning if promoted to an exception with an error handler.